cdap-ui/app/cdap/components/RulesEngineHome/RuleBook/RuleBook.scss

```diff
@@ -43,19 +73,56 @@ $version_label_color: gray;
   .rule-book-num-of-rules {
     margin-bottom: 14px;
   }
+  .rule-book-description {
+    font-size: 13px;
+    overflow: hidden;
+    position: relative;
+    width: 100%;
+    max-height: 4.5em;
+    margin-right: -10px;
+    padding-right: 10px;
+    word-break: break-word;
+
+    &:before {
+      content: '...';
```

Comment:
Its a good hack to start with but we might see some corner cases where the word is long enough that it goes to the newline in the middle of the sentence. This could create a gap between the place where the lastline ends and the ellipsis starts.

![screen shot 2017-08-17 at 2 25 30 pm](https://user-images.githubusercontent.com/1452845/29435789-3f8e4e74-835d-11e7-9cf0-7e7ba078aa48.png)

Something like this. We can improve on this later but for now can we add a `title` attribute to the div that shows it as a tooltip? We show the ellipsis but we don't show what is there when user hovers.